scripts/copper_bulk_submit.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+
+
+def grant_execute_permissions(directory_path) -> None:
+    """Grant execute permissions to all files in the given directory."""
+    # Construct the chmod command
+    command = ["chmod", "-R", "+x", directory_path]
+
+    try:
+        # Run the command
+        result = subprocess.run(command, check=True, text=True, capture_output=True)
+
+        # Print the command output
+        print("Permission command output:", result.stdout)
+    except subprocess.CalledProcessError as e:
+        # Print the error message
+        print("Error occurred while setting permissions:", e.stderr)
+
+
+def run_scp_command(local_path, remote_path) -> None:
+    """Run the scp command to copy files from local to remote."""
+    # Construct the scp command
+    command = ["scp", "-r", local_path, remote_path]
+
+    try:
+        # Run the command
+        result = subprocess.run(command, check=True, text=True, capture_output=True)
+
+        # Print the command output
+        print("SCP command output:", result.stdout)
+    except subprocess.CalledProcessError as e:
+        # Print the error message
+        print("Error occurred while transferring files:", e.stderr)
+
+
+if __name__ == "__main__":
+    # Define the paths
+    local_path = "./data"
+    remote_path = "ll720@login.hpc.cam.ac.uk:/rds/user/ll720/hpc-work"
```

Comment:
use an environmental variable, you can make vscode auto insert Env Variables !

It would also be good if you could make it call the necessary commands to submit the HPC job, and add a load_hpc_result function.

Also add these generic functions to pcc/hpc.py

Reply:
I think I need to discuss this with you tomorrow. I tried chatgpt but it is still bit confusing

Reply:
Yeah or just push what you have and I and do the environment variables for you